.github/workflows/unassignIssue.yml

```
@@ -0,0 +1,14 @@
+name: Unassign Issue After 7 days
+on:
+  schedule:
+    - cron: "0 0 * * *" ### Runs everyday at 00h
+
+jobs:
+  unassign-issues-labeled-waiting-for-contributor-after-1-days-of-inactivity:
+    name: Unassign issues labeled "Wartet auf Contributor" after 7 days of inactivity.
```

Qual o negocio dessa label aqui?
```suggestion
    name: Unassign issues labeled "Wartet auf Contributor" after 7 days of inactivity.
```

---

nada, esse name é exemplo da action na actions marketplace
achei que tivesse mudado, vou arrumar